src/styles/components/_index.scss

```diff
@@ -7,6 +7,7 @@
 @use "dropdown" as *;
 @use "form-control" as *;
 @use "input" as *;
+@use "navigation" as *;
```

Comment:
Mag dit niet mee in de Navigation.module?

Reply:
De css die daar in zit is voor de notification button die momenteel in `AppLayout` aangemaakt wordt. Ik denk niet dat het de bedoeling is om daar css modules te gebruiken?